internal/log-file-metric-exporter/Docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openshift/origin-release:golang-1.15 AS build
```

Comment:
This file is not required at all.  We modify the existing Dockerfile to copy the binary output to the desired location in the image.
* change to depend on building the new binary: https://github.com/openshift/cluster-logging-operator/blob/master/Makefile#L68
* copy new binary to /usr/bin like https://github.com/openshift/cluster-logging-operator/blob/master/Dockerfile#L34
* add a newly configured container to: https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/fluentd.go#L451
* probably need to add an endpoint so we can scrape the service https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/fluentd.go#L154

Reply:
@jcantrill  first change in Makefile will allow us to compile binary for log-file-metric-exporter, second change will copy this compiler binary to /usr/bin in CLO container. i am not able to comprehend how this gets executed part of CLO process. The third change - podspec for fluentd - "{Name: "entrypoint", VolumeSource: v1.VolumeSource{ConfigMap: &v1.ConfigMapVolumeSource{LocalObjectReference: v1.LocalObjectReference{Name: "fluentd"}}}}," , do we need to add one more entry point for log-file-metric-exported ?  fourth change also i need to better understand for exposing log_logged_bytes_total from log-file-metric-exporter.